recipes/easyquery/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "easyquery" %}
+{% set version = "0.1.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/easyquery-{{ version }}.tar.gz
+  sha256: de2872add2fa78c1b827523d124922426d85d37ca2a41041c809f8813b76a6c5
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - numexpr >=2.0
+    - numpy >=1.7
+    - python >=3.6
+
+test:
+  imports:
+    - easyquery
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/yymao/easyquery
+  summary: Create easy-to-use Query objects that can apply on NumPy structured arrays, astropy Table, and Pandas DataFrame.
+  license: MIT
+  license_file: LICENSE
```

Comment:
I opened a PR upstream (https://github.com/yymao/easyquery/pull/1) to include the license file in the source distribution. If you could add a comment like:

```yaml
# License file manually packaged. See: https://github.com/yymao/easyquery/pull/1
license_file: LICENSE
...
```

This will be a visual reminder to check the status of the license file upon future releases when updating the feedstock. If this PR gets merged or another solution is used, then once the license file is in the source distribution, you can remove the comment and delete the manually packaged copy.

Reply:
Thank you, @synapticarbors! I've added a comment to indicate this.